spec/plaid_spec.rb

```diff
@@ -122,4 +122,14 @@
       end
     end
   end
+
+  describe Plaid::Info do
```

Comment:
@justinworth do you happen to know the mystery behind `keys.yml`
I see it referenced in a couple of places, including here - https://github.com/Gusto/plaid-ruby/blob/master/spec/connect_spec.rb#L87
but don't see it in the repo